lib/fast_gettext/translation_repository/merge.rb

```diff
@@ -10,6 +10,9 @@ class Merge < Base
       def initialize(name, options={})
         clear
         super(name, options)
+        options.fetch(:chain, []).each do |repo|
```

Comment:
I think it does. You can create an empty repo and use `add_repo` later.
Also empty chain is still a chain and IMO it's usually simpler to accept empty input than protecting each usage with a condition.